modules/TestUtils.js

```diff
@@ -88,6 +94,14 @@ export const functionThrows = (fn, context, args, value) => {
 export const arrayContains = (array, value, compareValues) =>
   array.some(item => compareValues(item, value) !== false)
 
+/**
+ * Returns true if the given array contains the value, false
+ * otherwise. The compareValues function must return false to
+ * indicate a non-match.
+ */
+export const setContains = (set, value, compareValues) =>
+  arrayContains([ ...set ], value, compareValues)
```

Comment:
In that case, `Set.prototype.forEach.call(set, x => compareValues(value, x))` would be more efficient (but some engines don't have forEach, so you'd need to use `Function` and `for..of` there)